altbn128/buildAltbn128.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-go get && make clean && make linux
+go get && make clean && make linux-arm64
```

Comment:
don't we also need to make linux amd64 version? so that both versions for arm and amd are included in the final jar?

Reply:
Done